File: src/test/java/com/fish/KexieOpenJudgeApplicationTests.java
package com.fish;

import jakarta.annotation.Resource;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.security.crypto.password.PasswordEncoder;

@SpringBootTest
class KexieOpenJudgeApplicationTests {
    @Resource
    private PasswordEncoder passwordEncoder;
    @Test
    void contextLoads() {
        System.out.println(passwordEncoder.encode("qx310320"));
    }
}

File: src/test/kotlin/com/fish/KexieOpenJudgeApplicationTests.kt
package com.fish

import jakarta.annotation.Resource
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.security.crypto.password.PasswordEncoder

@SpringBootTest
internal class KexieOpenJudgeApplicationTests {
    @Resource
    private val passwordEncoder: PasswordEncoder? = null
    @Test
    fun contextLoads() {
        println(passwordEncoder!!.encode("qx310320"))
    }
}
